src/main/java/com/google/sps/servlets/DrinkRatingServlet.java

```diff
@@ -0,0 +1,142 @@
+// Copyright 2020 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     https://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package com.google.sps.servlets;
+
+import com.google.appengine.api.datastore.DatastoreService;
+import com.google.appengine.api.datastore.DatastoreServiceFactory;
+import com.google.appengine.api.datastore.Entity;
+import com.google.appengine.api.datastore.PreparedQuery;
+import com.google.appengine.api.datastore.Query;
+import com.google.appengine.api.datastore.Query.SortDirection;
+import com.google.gson.Gson;
+import java.io.IOException;
+import java.util.*;
+import javax.servlet.annotation.WebServlet;
+import javax.servlet.http.HttpServlet;
+import javax.servlet.http.HttpServletRequest;
+import javax.servlet.http.HttpServletResponse;
+
+// Servlet to process users rating a drink from a coffee shop
+@WebServlet("/drink-rating")
+public class DrinkRatingServlet extends HttpServlet {
+
+  private final String maxRating = "/5.0";
+  private List<String> knownDrinks = new ArrayList<>();
+  public Map<String, Integer> votes = createMap();
```

Comment:
What are the keys and values of this Map? Suggest a name like <key>To<value>Map e.g. userIdToEmailMap

Reply:
The keys are the ratings 1-5, and the values are the number of votes that drink got. I'll come up with a better variable name.

Reply:
@dag13 please address this